.gitignore

```diff
@@ -37,3 +38,5 @@ yarn-error.log*
 
 # jet brains
 .idea
+
+node_modules_/
```

Comment:
`node_modules_` is not a valid path I think, can be removed

`node_modules` already gets ignored on https://github.com/smlxl/evm.codes/pull/316/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L4